docs/administration/necessary-adjustments.md

```diff
@@ -0,0 +1,29 @@
+# Necessary adjustments
+
+Some adjustments that are necessary to represent the funding workflow of a specific giving organization can not be done via configuration pages.
```

Comment:
I'm wondering if it'd better to write what can/has to be done instead of what is not possible.

Reply:
I find that difficult in this context. The whole page exists with the purpose to state that there are some things that need adaptations on code level, namely funding case types.
It can be relevant for an organization to know about the things that are not configurable without the help of a developer. If everywhere only the things are mentioned that are possible, the necessary adjustments that are (currently) necessary for an organization to use this extensions are hidden between the lines. 
I am not very specific about what can and cannot be done (this might also change over time) but I find it important to mention the fact that adjustments are necessary.

Reply:
I've changed the text a bit. I guess it makes clear what I meant initially.